requirements.txt

```diff
@@ -13,6 +13,6 @@ ruamel.base==1.0.0        # via ruamel.yaml
 ruamel.ordereddict==0.4.9  # via ruamel.yaml
 ruamel.yaml==0.10.20      # via shub
 scrapinghub==1.7.0        # via shub
-shub==2.0.0
+shub==2.2.0
```

Comment:
Latest `shub` version redefines default endpoint, and it affects `shub-image` tests. Added the shub version requirement to `setup.py`, thanks